src/apps/backend/modules/account/internal/account_notification_preference_reader.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, cast
+from modules.account.internal.store.account_notification_preferences_model import AccountNotificationPreferencesModel
+from modules.account.internal.store.account_notification_preferences_repository import (
+    AccountNotificationPreferencesRepository,
+)
+from modules.notification.types import NotificationPreferences
+
+
+class AccountNotificationPreferenceReader:
+    @staticmethod
+    def get_notification_preferences_by_account_id(account_id: str) -> NotificationPreferences:
+        notification_preferences = AccountNotificationPreferencesRepository.collection().find_one(
+            {"account_id": account_id}
```

Comment:
Please make sure `account_id` is indexed on `AccountNotificationPreferencesRepository`, if not already, to make this query efficient.